src/scss/components/settings/_settings-panel.scss

```
@@ -25,7 +25,7 @@
   overflow-y: auto;
   padding: 0;
   position: absolute;
-  right: 1em;
+  right: max(1em, env(safe-area-inset-right));
```

What do you think about providing fallback values for `env` variables? So that in case the environment variable is not available, the `max` expression is still evaluated.

```suggestion
  right: max(1em, env(safe-area-inset-right, 0)); // env results 0 in case it is not available, so 1em is used
```

I've read about fallbacks for `env` here: https://developer.mozilla.org/en-US/docs/Web/CSS/env

---

[5504e88](https://github.com/bitmovin/bitmovin-player-ui/pull/744/commits/5504e88b421416fd2e79689aa8dd92ada66c545c)